dashboard/app/helpers/application_helper.rb

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
```

Comment:
Given that the default encoding in Ruby 2.0 and higher is `utf-8`, do we actually need these lines?

Reply:
Probably not. But I'm going to call that a job for another PR.

Reply:
Ah, is this now superseded by https://github.com/code-dot-org/code-dot-org/pull/23954?

Reply:
Yes, it is (the fixes).

After that PR gets merged, I'll reenable the rubocop.